tailscale/config.yaml

```diff
@@ -17,6 +17,12 @@ arch:
   - armv7
   - i386
 init: false
+map:
+  - ssl:rw
+options:
+  request_cert: false
+  certfile: fullchain.pem
+  keyfile: privkey.pem
```

Comment:
Maybe we can just default these and remove the config?

For example by storing them in `/ssl/tailscale/[fullchain|privkey].pem`?

Reply:
I'm using as @frenck says. IMHO it is safer to have a different folder for tailscale as you may still need the non-tailscale certificates for something else. You can define which TSL the services will use, by-service, instead of forcing all [working] services to use tailscale's and for some reason all breaks.

An use case I can provide is the inability to login to HA itself just because I was relying on tailscale, and the tailscale addon went down.

my 2 cents

Reply:
I'd like the option to choose, because I'm going to run HTTPS only on the tailscale address and firewall everything else. We could change the default to something else but i went with what other certificate add-ons are doing.

Reply:
> I'd like the option to choose

I'd like to reduce complexity and remove it.

Reply:
Fair enough. I'll remove it.